README.md

```diff
@@ -385,20 +390,15 @@ terraform apply -var 'workersNum=2' -var 'instanceType="t2.small"' \
 -var 'dockerRegistry="repo.url/image-name:latest"'
 ```
 
-Note that use of two environment variables for the AWS keys.
-Other variables defined in `var.tf` are passed as parameters.
-Once finished, you can access the newly created scheduler node by: `ssh -i ~/.aws/key.pem ec2-user@$(terraform output scheduler-info)`.
-For this to work, you might need to download a `pem` file from AWS console.
-In the cluster you can check the log at `/var/log/user-data.log`.
-You can also check the status of the running Docker containers using `docker ps`.
-Lastly, if everything went well, you should be able to access the web interface of the cluster.
+**Note** that use of two environment variables for the AWS keys. Other variables defined in `var.tf` are passed as parameters.
+
+<!-- could you restructure this paragraph a bit? --->
```

Comment:
Let's have another iteration on this paragraph after we merge your changes. OK?

Reply:
@Kacawi I meant this paragraph that you still want to edit. I will create a PR once we're done with this one.